src/services/redisCache.ts

```diff
@@ -0,0 +1,17 @@
+import { Redis } from "ioredis";
+import { REDIS_HOST, REDIS_PASSWORD, REDIS_PORT } from "../constants";
+
+const RedisCache = new Redis({
+  host: REDIS_HOST!,
```

Comment:
Why is REDIS_HOST potential nullable?

Reply:
Ohh good catch , its not needed, will fix